Abstract/portable-formula.rb

```diff
@@ -49,6 +58,21 @@ def archs
       return [Hardware::CPU.arch_32_bit]
     end
 
+    # On Tiger and Leopard, override the default behaviour.
+    # Normally we don't build 64-bit there, because the linker is
+    # temperamental, and so "universal" builds don't usually happen.
+    # However, for the purposes of distributing portable packages,
+    # it's very useful to be able to build i386/ppc binaries for use
+    # on both Intel and PowerPC Macs. The Apple-provided compilers
+    # are capable of this on both Intel and Mac hosts.
+    if MacOS.version < :snow_leopard
```

Comment:
You probably want to add `OS.mac?` as formulae in this repo should work on both mac and linux.